client/common/cmdline.c

```diff
@@ -1917,9 +1949,10 @@ int freerdp_client_settings_parse_command_line_arguments(rdpSettings* settings,
 		{
 			long type;
 			char* pEnd;
+			errno = 0;
 			type = strtol(arg->Value, &pEnd, 10);
 
-			if (errno != 0)
+			if ((errno != 0) && (errno != EINVAL))
```

Comment:
I wasn't aiming to make any functional changes here- just to make it work with a more POSIX-compliant (in this regard) libc.

I can make this change if you'd like, though- the original submitter of the previously mentioned issue had a patch that also validated type before it was later passed to a function in case it failed both numeric parsing and wasn't one of the present values, but I had intentionally left that out as I considered it ESCOPE for the moment.